src/blockchain/rpc.py

```diff
@@ -0,0 +1,63 @@
+import requests
+import json
+import random
+
+url = "http://user:password@bitcoind:8332"
+
+
+def execute_rpc(rpc):
```

Comment:
```suggestion
def execute_rpc(rpc):
```
I would move this method to RPC class as "RPC.execute"

Reply:
Done